generated/nidcpower/nidcpower.proto

```diff
@@ -473,6 +473,53 @@ enum TriggerType {
   TRIGGER_TYPE_NIDCPOWER_VAL_SOFTWARE_EDGE = 1015;
 }
 
+message NIComplexNumber {
```

Comment:
Aren't messages typically `*Request` and `*Response`. How comes these new ones don't have that?

Reply:
Adding to Rohan's comment. Should these messages be generated only after an rpc API is added?

Reply:
Custom types will not have request and response as they are not rpc APIs. For reference, you can look at the scope proto file [here](https://github.com/doshirohan/grpc-device/blob/main/generated/niscope/niscope.proto).
We can add the rpc APIs that use these custom types only after we have the custom types defined in config.py. Those APIs will be a part of subsequent PRs as they need to be custom implemented.

Reply:
Looking through the proto file you linked, essentially these are types used within request/response messages - apart from primitive types supported by gRPC. Is that correct?

Reply:
yeah

Reply:
@ganeshnie I have added the rpc APIs also in this PR now